apps/els_lsp/src/els_completion_provider.erl

```diff
@@ -412,9 +406,11 @@ completion_item(#{kind := Kind, id := {F, A}}, true)
    , kind             => completion_item_kind(Kind)
    , insertTextFormat => ?INSERT_TEXT_FORMAT_PLAIN_TEXT
    };
-completion_item(#{kind := Kind, id := Name}, _)
-  when Kind =:= record;
-       Kind =:= define ->
+completion_item(#{kind := Kind = record, id := Name}, _) ->
+  #{ label            => atom_to_label(Name)
+   , kind             => completion_item_kind(Kind)
+   };
+completion_item(#{kind := Kind = define, id := Name}, _) ->
   #{ label            => atom_to_binary(Name, utf8)
```

Comment:
Shouldn't we also use `atom_to_label/1` here?

Reply:
I think that would give you suggestions in the form of `?'MACRO'`, which isn't quite right. Initially I did what you're suggesting here, and that broke a couple of testcases.

Reply:
That makes sense!